src/css/_gaps.scss

```diff
@@ -0,0 +1,21 @@
+.markdown > h1,
```

Comment:
1. Please move that to the typography file, next to the styles of h1,h2, etc.
2. Use nesting